packages/vite/src/node/constants.ts

```diff
@@ -133,7 +133,7 @@ export const KNOWN_ASSET_TYPES = [
   'webp',
   'avif',
   'cur',
-
+  'jxl',
```

Comment:
```suggestion
  'jxl',

```
Let's keep the empty line.